rakefile.rb

```diff
@@ -38,32 +38,42 @@
 approval_tests = "tests/FakeItEasy.Tests.Approval/bin/Release/FakeItEasy.Tests.Approval.dll"
 
 repo = 'FakeItEasy/FakeItEasy'
+ci_server_url = 'http://teamcity.codebetter.com/admin/editBuildParams.html?id=buildType:bt929'
 release_issue_labels = ['P2', 'build', 'documentation']
-release_issue_body = <<-eos
-**Ready** when all other issues on this milestone are **Done** and closed.
+release_issue_common_steps = <<-eos
+**in-progress** when all other issues on this milestone are closed.

```

Comment:
Not sure I agree. It's **ready** when the other issues are closed, but it won't be **in-progress** until someone picks up the task and does it.